utils/serverStatistic.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const { default: fetch } = require("node-fetch");
+
+const application_id = 'd78e7f67147305f3042bef05755fa168';
+const MAX_ACCOUNTS_PER_REQUEST = 100;
+const TOTAL_ACCOUNTS = 594859325;
+const BASE_URL = 'https://api.wotblitz.eu/wotb';
+const allPlayerStats = { account: {}, tanks: [] };
+
+const statsNames = ['battles', 'damage_dealt', 'damage_received', 'frags', 'spotted', 'wins', 'losses', 'survived_battles'];
+const achievementsName = ['markOfMastery', 'markOfMasteryI', 'markOfMasteryII', 'markOfMasteryIII'];
+const getStatsObject = (statsName, stats) => Object.fromEntries(statsName.map((statName) => [statName, stats[statName] || 0]));
+
+const getInfo = async (url) => {
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data.data;
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
+const getVehicleStats = async (accountId) => {
+  const result = {};
+  const vehicleStatsURL = `${BASE_URL}/tanks/stats/?application_id=${application_id}&account_id=${accountId}`;
+  const vehicleAchievmentsURL = `${BASE_URL}/tanks/achievements/?application_id=${application_id}&account_id=${accountId}`;
+  const vehicleStats = await getInfo(vehicleStatsURL);
+  const vehicleAchievments = await getInfo(vehicleAchievmentsURL);
+
+  const stats = vehicleStats[accountId];
+  const achievements = vehicleAchievments[accountId];
+
+  if (!stats || !achievements) return;
+
+  stats.map((tankStats) => {
+    const battleLifeTime = tankStats.battle_life_time;
+    if (!battleLifeTime) return;
+    const regular = getStatsObject(statsNames, tankStats.all);
+    result[tankStats.tank_id] = { regular, battleLifeTime };
+  });
+
+  achievements.map((tankAchivements) => {
+    if (!result[tankAchivements.tank_id]) return;
+    const mastery = getStatsObject(achievementsName, tankAchivements.achievements);
+    result[tankAchivements.tank_id].mastery = mastery;
+  });
+
+  return Object.entries(result).map(([wotId, data]) => ({ wotId, ...data }));
+};
+
+const fetchPlayerStats = async (playerIds) => {
+  const playerStatsURL = `${BASE_URL}/account/info/?application_id=${application_id}&account_id=${playerIds.join(',')}&extra=statistics.rating`;
+  const playerAchievmentsURL = `${BASE_URL}/account/achievements/?application_id=${application_id}&account_id=${playerIds.join(',')}`;
+
+  try {
+    const playerStats = await getInfo(playerStatsURL);
+    const playerAchievments = await getInfo(playerAchievmentsURL);
```

Comment:
By using `Promise.all`, both `getInfo` functions will be executed in parallel, and `await` will wait for both promises to resolve. This approach can potentially reduce the overall execution time compared to awaiting them one after another.
```suggestion
  const [playerStats, playerAchievments] = await Promise.all([getInfo(playerStatsURL), getInfo(playerAchievmentsURL)]);
```